upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
+# Create the CRDs by applying the crds.yaml manifest.
+echo "Creating Calico CRDs..."
+/sbin/kubectl apply -f crds.yaml
+
+# Get IPPools list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+echo "Migrating IPPools..."
+/sbin/calicoctl-v1.4 get ippool -o yaml > ippool.yaml
+retval=$?
+if [ $retval == 0 ];
+then
+    if [ -f ippool.yaml ]; then
+        echo "Successfully got the IPPools:"
+        cat ippool.yaml
+        /sbin/calicoctl-v1.5 apply -f ippool.yaml
+    else
+       echo "No IPPools found to migrate."
+    fi
+else
+    echo "Failed to get IPPools"
```

Comment:
hmm I'm thinking we should do the best effort migration, so if get IPPool fails, we still try to migrate other resources instead of exiting right away.